scripts/zk_benchmark.py

```diff
@@ -0,0 +1,290 @@
+from typing import Generator
+from kazoo.client import KazooClient
+from tqdm import tqdm
+from time import perf_counter
+from random import randbytes
+import string
+import random
+import concurrent.futures
+
+# typedef our generator
+NodeGenerator = Generator[tuple[str, bytes], None, None]
+
+# Util functions
+def random_string(size: int) -> str:
+    letters = string.ascii_letters + string.digits + "_"
+    res = "".join(random.choices(letters, k=size))
+    return res
+    
+def randstr(low: int, high: int) -> str:
+    size = random.randrange(low, high)
+    return random_string(size)
+
+# Generator which generates the child node data based on params
+def node_generator(num: int, name="", min_name_len=-1, max_name_len=-1, data_size_bytes=1024) -> NodeGenerator:
+    gen = 0
+    while gen < num:
+        name_res = ""
+        if len(name) > 0:
+            name_res = "{}_{}".format(name, str(gen + 1))
```

Comment:
this wouldn't take care of name length requirements, but that should be fine in case fixed name is needed. just add a comment for the same.